schema/filter_artworks.js

```diff
@@ -28,6 +30,16 @@ export const FilterArtworksType = new GraphQLObjectType({
       type: GraphQLInt,
       resolve: ({ aggregations }) => aggregations.followed_artists.value,
     },
+    merchandisable_artists: {
+      type: new GraphQLList(Artist.type),
+      description: 'Returns a list of merchandisable artists (sorted by merch score), if the aggregation was requested.',
+      resolve: ({ aggregations }) => {
+        if (!isExisty(aggregations.merchandisable_artists)) {
+          return null;
+        }
+        return gravity(`artists`, { ids: keys(aggregations.merchandisable_artists) });
```

Comment:
https://github.com/artsy/gravity/pull/10228 fetch all artists by id, maintaining the order